### DISCUSSION TITLE
Use of IsPost: false

### DISCUSSION THREAD

**Discussion by CharliePoole** [statiqdev/Discussions#33] [Q&A] [answered]
Not sure if I changed something in my site or if something changed underneath me. :smile: 

It seems to me that I was originally able to have a non-post page in one of my PostSource directories by including `IsPost: false` in the front matter of that file. It appears to no longer be working. Have others run into this?

Unfortunately, I had stopped needing this feature so I was unaware of any problem until just today, when I need it again. Before I start backing all my (supposedly) unrelated changes, I'm hoping somebody will have a hint for me.

UPDATE: I'm using CleanBlog btw, with changes made in my input directory.

**Comment by daveaglick** [comment, marked as answer]
As far as I can tell, `IsPost` should still be working. Note that it only controls whether the post or non-post header and footer should be shown in the page layout - it doesn't control whether a page is actually considered a blog post or not.

What's the goal - do you have a page in Markdown or Razor inside one of your `PostSource` folders that you want to render but not treat as a blog post? If so, I don't think there's support for that right now but it would be trivial to add to the theme.

**Comment by CharliePoole** [comment]
Yes, now that you mention it, that's exactly what I'm observing. The IsPost setting works fine controlling __how__ the page itself is rendered. I made the assumption that it should have the effect of filtering what appears in summary listings. I can see how to fix it if I need to now.

As you guessed, this only comes up because I was placed a non-post pages in the same directory with posts because it seemed like it should work and was convenient. It's a toss-up whether I should make this work or just take the usual doctor's advice and "stop doing that." :smile:

**Reply by daveaglick** [reply]
I think your initial assumption was the correct one - it makes total sense that a value called `IsPost` should filter a page out of the blog postings if set to false. And having non-post pages mixed in with posts in the same folder also seems like a totally reasonable thing you might want to do.

Going to try adding a quick filter to the post archive file(s), I think that's really all we'll need.

**Reply by daveaglick** [reply]
Yep, that worked! Checking it in now - if you update CleanBlog you should get the changes. Otherwise if you've made a bunch of changes and customized the theme and don't want to merge, you can take a look at the commit - it didn't take a ton of additions.

**Reply by CharliePoole** [reply]
Thanks, that looks good. I can easily go either way.
